.drone.star

```diff
@@ -1865,7 +1880,7 @@ def docs():
                         "from_secret": "github_token",
                     },
                     "pages_directory": "docs/hugo/content",
-                    "target_branch": "docs",
+                    "target_branch": "docs-stable-2.0",
```

Comment:
@mmattel @dragonchaser This is my proposed branch name for the stable-2.0 docs.

Tagging needs an extra pull request.